tests/statement_serializer_tests/schema/table.cpp

```diff
@@ -0,0 +1,33 @@
+#include <sqlite_orm/sqlite_orm.h>
+#include <catch2/catch_all.hpp>
+
+using namespace sqlite_orm;
+
+TEST_CASE("statement_serializer table_t") {
```

Comment:
this test case is not related to PRs topic but I wanted to add this last time in the next PR so here we are